dbld/images/required-packages/artful-obs.txt

```diff
@@ -0,0 +1,2 @@
+criterion-dev
```

Comment:
Considering the order of package install location is: pip/obs/distro, is it possible to create common files? Do we need separate list for artful-obs, xenial-obs etc?

Reply:
@lbudai Can you help with obs? Is it possible to give different names for the same package on different platforms? (I know from experience that it is allowed by the Ubuntu maintainers to change package names from distribution to distribution. It is possible with obs?)